### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 94noni**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | yes as doc
| Tickets       | see desc
| License       | MIT
| Doc PR        | .

Related to https://packagist.org/packages/symfony/web-server-bundle/stats
Also I do have a question, do Symfony uses the https://getcomposer.org/doc/04-schema.md#abandoned feature?
It can be useful I think

I’ve seen many differents way, inside the repo readme, via github repo description with [DEPRECATED], even marking the repo as readonly
IMHO a similar aproach should be done for all packages of the organisation :)

### CODE REVIEW

File: src/Symfony/Bundle/WebServerBundle/README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 WebServerBundle
 ===============
 
+**CAUTION**: this bundle is deprecated since Symfony 4.4. Instead, use the
+[Symfony Local Web Server](https://symfony.com/doc/current/setup/symfony_server.html).
+
+-----
```
Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/README.md]: This line and the next one should be removed

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/WebServerBundle/README.md (hunk 1 above)
Comment (94noni) [reply src/Symfony/Bundle/WebServerBundle/README.md]: Ok I will do this 👍🏻
FYI I took inspiration at https://github.com/symfony/inflector

**Comment by wouterj** [comment]
>  Also I do have a question, do Symfony uses the https://getcomposer.org/doc/04-schema.md#abandoned feature?

Yes, we are. However, it's not abandoned yet (the reason you can make this PR is because we still maintain the 4.4 version). It'll receive this flag once 4.4 reaches end of life.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @94noni.

**Pull request merged**